trinity/chains/light.py

```diff
@@ -59,7 +59,7 @@
 )
 
 if TYPE_CHECKING:
-    from eth.vm.base import BaseVM  # noqa: F401
+    from eth.vm.base import VM  # noqa: F401
```

Comment:
Another case where we could stay on `BaseVM` if we just ensure we create that abstract method